website/pages/en/index.html

```diff
@@ -1,9 +1,10 @@
 <!doctype html>
-<html lang="en-gb" class="no-js">
+<html lang="en-gb" class="no-js" xmlns="http://www.w3.org/1999/xhtml"
+      xmlns:fb="http://ogp.me/ns/fb#">
 <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
-    <meta property="og:image"              content="img/sumo-share.png" />
+    <meta property="og:image"              content="https://www.sumochallenge.org/img/sumo-share.png" />

```

Comment:
Our site is sumochallenge.org, not www.sumochallenge.org.  But I believe that in docusaurus, you should use the relative pathname for referencing the internal documents, not the full pathname.

Reply:
The categories and xsd files are in the new sample_data.zip file along with a README explaining all the files. The pull request updates the sample data as well as the preview image of the website which currently is the invisible SUMO logo. I updated it to a snapshot of the scenes at the front page of the website, so when the page is shared, that will be displayed instead.

Reply:
I am going ahead and merging the pull request, but I think it will be better to keep the xsd and categories list separate from the sample data.  The sample data is optional, so some people may not download it and will not be aware of the information.  Another option is to incorporate it into the API.